pac4j-http/src/main/java/org/pac4j/http/credentials/CredentialUtil.java

```diff
@@ -0,0 +1,61 @@
+package org.pac4j.http.credentials;
+
+import java.security.MessageDigest;
+import java.security.NoSuchAlgorithmException;
+
+/**
+ * This class contains utility methods related to credential encryption algorithms
+ * (MD5 for http digest)
+ *
+ * @author Mircea Carasel
+ */
+public class CredentialUtil {
```

Comment:
There is: org.apache.commons.codec.binary.Hex;... and there is already a dependency. I will use it instead my manual conversion